gsf-core/src/main/java/tools/gsf/navigation/Node.java

```diff
@@ -46,4 +46,24 @@
      * @return this node's children, never null.
      */
     List<NODE> getChildren();
+    
+    /**
```

Comment:
Nope. They are the only reason it is possible to bind the nodes together in a "deferred" way.

SitePlanNavService.createAssetNode (abstract) creates and populates the AssetNode instance but it does NOT set its parent / children.

That's done by SitePlanNavService's constructor, once it has instantiated all nodes (via createAssetNode)